### DISCUSSION TITLE
Intel x86 Atom_64 System Image Failed: This download could not be finalized from the interim state. Retrying without caching.

### DISCUSSION THREAD

**Discussion by Temp-insta** [community/community#20930] [Programming Help] [answered]
I was trying to install Intel x86 Atom_64 System Image from the SDK manager of Android Studio. But the following error came:
```sh
This download could not be finalized from the interim state. Retrying without caching.
```
The actual log looks something like this:
```sh
Packages to install: - Intel x86 Atom_64 System Image (system-images;android-31;default;x86_64)


Preparing "Install Intel x86 Atom_64 System Image (revision: 3)".
Downloading https://dl.google.com/android/repository/sys-img/android/x86_64-31_r03.zip
This download could not be finalized from the interim state. Retrying without caching.
Downloading https://dl.google.com/android/repository/sys-img/android/x86_64-31_r03.zip
Failed packages:
- Intel x86 Atom_64 System Image (system-images;android-31;default;x86_64)
```
How can I fix it?

**Comment by santosharron** [comment, marked as answer]
Try to download it manually using the link:
```sh
https://dl.google.com/android/repository/sys-img/android/x86_64-31_r03.zip
```
After downloading extract it to the SDK dir so the resulting path looked like this:
```sh
AppData\Local\Android\Sdk\system-images\x86_64
```
And Then restart your Android Studio, Then it will work :)

**Reply by syedrizvi** [reply]
Was facing the same issue. @santosharron your solution fixed my error. And now it shows as installed in the SDK manager.

**Reply by AntonioLi1** [reply]
Thanks for the answer @santosharron!
I did exactly what you said but my SDK manager still says it is not installed.
Do you have any advice for me?

**Reply by dutrasr** [reply]
@AntonioLi1, I think there is a few folders missing on that path. I extracted on the following path and it worked:
```
Android/Sdk/system-images/android-31/intel/x86_64
```

**Reply by oyooyo** [reply]
I came upon this problem when following react-native's installation instructions.
Moving the downloaded & extracted x86_64 system image to `AppData\Local\Android\Sdk\system-images\x86_64` did not make it show up under "Android 12.0 (S)" as suggested in react-native's installation instructions.

When I moved it to
```
Android/Sdk/system-images/android-31/intel/x86_64
```
instead as @dutrasr suggested, it worked fine. Thanks!

**EDIT**: I just saw that android studio actually creates a warning log message, suggesting that the correct path is actually
```
Android/Sdk/system-images/android-31/default/x86_64
```

**Reply by sleekLancelot** [reply]
Hi, is there an equivalent of this for AMD systems.

**Reply by wonsuchai** [reply]
Android/Sdk/system-images/android-31/default/x86_64

**Reply by mucahitdev** [reply]
Android/Sdk/system-images/android-31/default/x86_64
great

**Reply by bartoS2ostrokolowicz** [reply]
if anyone has a problem with new Android Studio Giraffe, this path worked for me 

Android/Sdk/system-images/android-31/google_apis/x86_64

**Reply by charith-codex** [reply]
2024 working path is :

>AppData\Local\Android\Sdk\system-images\android-34\google_apis_playstore\x86_64

**Reply by Justatnann** [reply]
Thank you for the advice!! help a lot

**Reply by saadrahman8** [reply]
> if anyone has a problem with new Android Studio Giraffe, this path worked for me
> 
> Android/Sdk/system-images/android-31/google_apis/x86_64

should extract the 86-64-35-r7 into the 86-64?

**Reply by miltonbita** [reply]
novo caminho para dezembro de 2024 

AppData\Local\Android\Sdk\system-images\android-35\google_apis_playstore\x86_64

**Reply by miltonbita** [reply]
> 2024 working path is :
> 
> > AppData\Local\Android\Sdk\system-images\android-35\google_apis_playstore\x86_64

**Reply by jorginhopc** [reply]
new download link for API 35, revision 9

https://dl.google.com/android/repository/sys-img/android/x86_64-35_r09.zip

**Comment by hqasraful** [comment]
Same problem happened to me. After two days constant trying I solved the problem. I use internet from my mobile using **JIO SIM**. I just switched my alternate mobile for internet data where I am also using **VODAFONE SIM**. You just switch from your current internet to another internet even over **WIFI**. I see in some cases some websites does not work properly in Jio network otherwise I prefer this network(**JIO**) for its top-notch speed and services. If in your case whatever your data provider just try this method use other data provider or WIFI. If this method solves your problem just hit the like button so others can get help. Thanks

**Reply by SatyShrey** [reply]
Thank you so much for this solution

**Comment by pptriana8986** [comment]
estoy en cuba y no puedo descargar https://dl.google.com/android/repository/sys-img/android/x86_64-35_r09.zip desde ese sitio. que hago para descargar la imagen?
